lighthouse-cli/test/smokehouse/frontends/smokehouse-bin.js

```diff
@@ -58,6 +58,18 @@ function getDefinitionsToRun(allTestDefns, requestedIds, {invertMatch}) {
     console.log(usage);
   }
 
+  if (onlyUrls) {
+    smokes = smokes.filter(smoke => {
+      smoke.expectations = smoke.expectations.filter(expectation => {
+        const url = expectation.lhr.requestedUrl;
+        const shouldRunUrl = onlyUrls.some(pattern => new RegExp(pattern).test(url));
```

Comment:
> `new RegExp(pattern)`

it seems like this would be a problem with some of the gnarlier URLs when c/p verbatim

Reply:
> > `new RegExp(pattern)`
> 
> it seems like this would be a problem with some of the gnarlier URLs when c/p verbatim

actually they look mostly ok, it's just the test urls with querystrings that break it, since `?` won't match itself in `new RegExp(urlWithQuery).test(urlWithQuery)`

(are there other metacharacters that aren't usually url encoded that could make their way into one of our test URLs?)

Reply:
I asked my friend SO for help.